doc/sphinx-guides/source/installation/installation-main.rst

```diff
@@ -36,16 +44,24 @@ The script will prompt you for some configuration values. If this is a test/eval
 - Rserve User Name: rserve
 - Rserve User Password: rserve
 
+**New, as of 4.3:**
+
+- Administration Email address for the installation;
+- Postgres admin password - We'll need it in order to create the database and user for the Dataverse to use, without having to run the installer as root. If you don't know your Postgres admin password, you may simply set the authorization level for localhost to "trust" in the PostgresQL ``pg_hba.conf`` file (See the PostgresQL section in the Prerequisites). If this is a production evnironment, you may want to change it back to something more secure, such as "password" or "md5", after the installation is complete.
+- Network address of a remote SOLR search engine service (if needed) - In most cases, you will be running your SOLR server on the same host as the Dataverse application (then you will want to leave this set to the default value of ``LOCAL``). But in a serious production environment you may set it up on a dedicated separate server. 
+- The URL of the TwoRavens application GUI, if this Dataverse node will be using a companion TwoRavens installation. Otherwise, leave it set to ``NOT INSTALLED``. 
+
+
 The script is to a large degree a derivative of the old installer from DVN 3.x. It is written in Perl. If someone in the community is eager to rewrite it, perhaps in a different language, please get in touch. :)
 
 All the Glassfish configuration tasks performed by the installer are isolated in the shell script ``dvinstall/glassfish-setup.sh`` (as ``asadmin`` commands). 
 
-As the installer finishes, it mentions a script called ``post-install-api-block.sh`` which is **very important** to execute for any production installation of Dataverse. Security will be covered in :doc:`config` section but for now, let's make sure your installation is working.
+**IMPORTANT:** Please note, that "out of the box" the installer will configure the Dataverse to leave unrestricted access to the administration APIs from (and only from) localhost. Please consider the security implications of this arrangement (anyone with shell access to the server can potentially mess with your Dataverse). An alternative solution would be to block open access to these sensitive API endpoints completely; and to only allow requests supplying a pre-defined "unblock token" (password). If you prefer that as a solution, please consult the supplied script ``post-install-api-block.sh`` for examples on how to set it up.
```

Comment:
If we want, we **could** note that this is consistent with Solr (see http://guides.dataverse.org/en/4.2.4/installation/prerequisites.html#securing-solr ) ... Solr allows you delete data from localhost if you can ssh onto the box. I think this is secure enough for both Dataverse and Solr. It's good that you explain how to further lock things down though, @landreev .